extension-manifest-v3/src/store/tab-stats.js

```diff
@@ -14,7 +14,9 @@ import { store } from 'hybrids';
 const Tracker = {
   id: true,
   name: '',
-  requests: [{ url: '', blocked: false }],
+  requests: [{ url: '', blocked: false, modified: false }],
+  requestsBlocked: ({ requests }) => requests.filter((r) => r.blocked),
```

Comment:
Move computed properties to the end of the map. It splits visually what is a part of the source data, and what is a computed local data. It is done in that way for `Stats`.